### DISCUSSION TITLE
How to build static libuv on solaris 8 and OpenVMS?

### DISCUSSION THREAD

**Discussion by serameo** [libuv/libuv#3758] [Q&A]
Currently, I could not follow the instruction on the first page.

**Comment by bnoordhuis** [comment]
I have no idea but my best guess is that it's not going to compile, let alone work. Libuv has never been ported to VMS (whether open or closed) and Solaris 8 has been EOL for 10 years now. Solaris 11 may work but 8? Probably not.

**Reply by serameo** [reply]
Thanks for your answer

**Comment by skissane** [comment]
Jake Hamby has an experimental port of libuv to OpenVMS here: https://github.com/jhamby/vms-libuv

Never used OpenVMS so can't comment on how well it works
